FeedAPIChallenge/Feed API/FeedImagesMapper.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  FeedImagesMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Aleksandr Honcharov on 01.06.2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+enum FeedImagesMapper {
+	private struct Root: Decodable {
+		let items: [Image]
+
+		var feed: [FeedImage] {
+			return items.map { $0.feedImage }
+		}
+	}
+
+	private struct Image: Decodable {
+		let id: UUID
+		let description: String?
+		let location: String?
+		let url: URL
```

Comment:
These properties are not used outside of `Image`, so they can be private.
```suggestion
		private let id: UUID
		private let description: String?
		private let location: String?
		private let url: URL
```